### ISSUE TITLE
logR and copy.number in cna.seg. files

### ISSUE THREAD

**Issue by daguilarvhio** [broadinstitute/ichorCNA#104]
Dear all,

I am struggling to understand some of the data in the *cna.seg files output, particularly how the "copy.number" column if derived from the "logR" column.

I understand that the copy number is calculated as:

No loss/gain= log2(2/2) = 0
One copy gain = log2(3/2) = 0.57
Two-copy gain = log2(4/2) = 1
One-copy loss = log2(1/2) = -1

(As mentioned [earlier](https://github.com/broadinstitute/ichorCNA/issues/68) in this forum).

However, this is not what I see in the  *cna.seg files output. For instance:

<html>
<body>

chr | start | end | copy.number | event | logR | subclone.status | Corrected_Copy_Number | Corrected_Call | logR_Copy_Number
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
6 | 28500001 | 29000000 | 5 | HLAMP | -9.0929 | 0 | 5 | HLAMP | 0.015625
6 | 29000001 | 29500000 | 5 | HLAMP | -10.0687 | 0 | 5 | HLAMP | 0.015625
6 | 29500001 | 30000000 | 5 | HLAMP | -4.5952 | 0 | 5 | HLAMP | 0.015625
6 | 30000001 | 30500000 | 5 | HLAMP | -10.031 | 0 | 5 | HLAMP | 0.015625
6 | 30500001 | 31000000 | 5 | HLAMP | NA | 0 | 5 | HLAMP | NA
6 | 31000001 | 31500000 | 5 | HLAMP | -7.8582 | 0 | 5 | HLAMP | 0.015625
6 | 31500001 | 32000000 | 5 | HLAMP | NA | 0 | 5 | HLAMP | NA
6 | 32000001 | 32500000 | 5 | HLAMP | NA | 0 | 5 | HLAMP | NA
6 | 32500001 | 33000000 | 5 | HLAMP | -8.9581 | 0 | 5 | HLAMP | 0.015625
6 | 33000001 | 33500000 | 5 | HLAMP | -3.2627 | 0 | 5 | HLAMP | 0.015625
</body>
</html>

How is it possible that a logR=-9.0929 generates to a copy.number=5?

Another example:

<html>
<body>

chr | start | end | copy.number | event | logR | subclone.status | Corrected_Copy_Number | Corrected_Call | logR_Copy_Number
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
4 | 500001 | 1000000 | 0 | HOMD | -0.0322 | 0 | 0 | HOMD | 0.015625
4 | 1500001 | 2000000 | 0 | HOMD | 0.0093 | 0 | 0 | HOMD | 8.852849318524
4 | 2000001 | 2500000 | 0 | HOMD | 0.0695 | 0 | 0 | HOMD | 42.7650489610307
</body>
</html>

Shouldn't a logR ~0 mean that copy.number=2, instead of copy.number=0?

Am I missing something in the way that copy.number is calculated?

**Comment by SuqinYang** [comment]
@daguilarvhio 
Hi,have you solved the problem? This question confuses me too.
